src/classes/jsonapi-route.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
```

Comment:
Is this zombie code? Or there are no tests - The serialize method looked suspiciously incomplete here, so I cmd+f searched for `json-api-route.js` on this page but couldn't find any instances of it being used.